tests/end2end/test-format-r.R

```diff
@@ -0,0 +1,3 @@
+paths <- commandArgs(trailingOnly = TRUE)

```

Comment:
Similar to `shiny.benchmark`, we can use this script, if ever we need to expand the tests on `format_r()` to check the results of different directories.

Reply:
I specified default paths to `app` and `tests`.